lib/auth.coffee

```diff
@@ -240,10 +249,9 @@ getAuth = (deps, opts) ->
 	exports.getToken = (callback) ->
 		auth.getKey()
 		.then (savedToken) ->
-			throw new errors.ResinNotLoggedIn() if not savedToken?
```

Comment:
Why don't we need this line? Does `getKey` throw for us if it's undefined?

Reply:
That will but not only when it's null or undefined but also when it's incorrect because of other reasons (eg. token set to a number instead of string). In such case the ResinMalformedToken error will be thrown.

Reply:
I edited the commits. Do you think it's ok now?